src/canisters/frontend/ic-certified-assets/src/http.rs

```diff
@@ -155,4 +188,20 @@ impl HttpResponse {
             streaming_strategy: None,
         }
     }
+
+    pub fn build_redirect(status_code: u16, location: String) -> HttpResponse {
+        if ![300, 301, 302, 303, 304, 307, 308].contains(&status_code) {
+            return HttpResponse::build_400(&format!(
+                "incorrect asset redirect configuration: response_code \"{}\" is not valid HTTP response code",
+                status_code
+            ));
+        }
```

Comment:
This check seems unnecessary, since the only value passed is 308.  For this PR, I suggest removing the parameter altogether and renaming this function `build_permanent_redirect`.

If there is a later PR that allows for configurable status codes, I would suggest to validate the values when reading the configuration, and _maybe_ in the canister when setting the configuration, rather than when processing requests.